components/typography/stories/template.js

```diff
@@ -21,6 +21,7 @@ export const Template = (args = {}, context = {}) => {
 		customClasses = [],
 		customStyles = {},
 		skipLineBreak = false,
+		lang = "",
```

Comment:
You can fetch the global lang from the context object like this and use it as a fallback if you like for when the block doesn't define it's own lang property.

```
	const { globals = {} } = context;
	const lang = globals.lang ?? "en-US";
```